LayoutTests/http/wpt/webauthn/public-key-credential-create-failure.https-expected.txt

```diff
@@ -7,6 +7,8 @@ CONSOLE MESSAGE: User gesture is not detected. To use the WebAuthn API, call 'na
 
 PASS PublicKeyCredential's [[create]] with timeout
 PASS PublicKeyCredential's [[create]] with a mismatched RP ID
+PASS PublicKeyCredential's [[create]] with a empty string RP ID
+PASS PublicKeyCredential's [[create]] with a null RP ID
```

Comment:
Will we eventually get 'LayoutTests/imported/w3c/web-platform-tests' to include the web authentication tests? Or are they all being imported to http? I'm not sure what our policy is on where WPT tests live.

Reply:
I've been investigating using 'LayoutTests/imported/w3c/web-platform-tests' but haven't yet been successful in getting the tests to run within run-webkit-tests after import.